src/components/common/layout.js

```diff
@@ -16,6 +16,13 @@ import Header from "./header";
 const globalStyles = css`
   :root {
     --midnight: #2d3142;
+    --white: #ffffff;
+    --taupe: #f7f5f4;
+    --cobalt: #3772ff;
+    --light-grey: #d3d3d3;
+    --yellow: #ffba08;
+    --denim: #424b71;
+    --error: #b5261d;
```

Comment:
It's interesting that they're all colour names except for `error`.

If ever we were to do something like a dark colour mode, we'd be able to reuse `error` by overriding its values, but in all the other cases, we'd have to change which variable is used, everywhere it's used, negating one of the benefits. I wonder if we can work with @carinapope13 to give all these colours purposeful names.